react/week4/github-user-searcher/src/components/UsersSearcher.jsx

```diff
@@ -0,0 +1,40 @@
+import React, { useContext } from 'react';
+import DisplayUser from './DisplayUser.jsx';
+import {UserContext} from './UserSearcherProvider.jsx';
+
+function UsersSearcher(){
+    const ctxObj = useContext(UserContext);
```

Comment:
This ctxObj is a bit scripted variable name. I would name it something more descriptive.

Reply:
You are completely right. I made it temporary but I forgot to change it.